### PR TITLE
Tox travis that recommended by jazzband

### PR DESCRIPTION
**Pull request by shimizukawa**
https://jazzband.co/about/releases#travis

### Purpose
- easy maintenance

### Relates
- run result on travis: https://travis-ci.org/jazzband/django-redshift-backend/builds/447104819

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -7,6 +7,17 @@ envlist =
...
+[travis]
+python =
+    2.7: py27
+    3.5: py35
+    3.6: py36,flake8,readme
+
+[travis:env]
+DJANGO =
+    1.11: dj111
+    2.0: dj20
```
Comment (shimizukawa) [inline tox.ini]: travis and travis:env sections come from:
https://tox-travis.readthedocs.io/en/stable/envlist.html

File: .travis.yml [hunk 2]
```diff
@@ -1,24 +1,20 @@
...
-    python: 3.6
-  - env: TOX_ENV=readme
-    python: 3.6
-install:
-- pip install -U pip setuptools tox
-script:
-- tox -e "${TOX_ENV}"
+    - env: TOXENV=flake8,readme
+      python: 3.6
+install: pip install tox-travis
```
Comment (shimizukawa) [inline .travis.yml]: tox-travis is recommended by jassband
https://jazzband.co/about/releases#travis

**Comment by jezdez** [comment]
I'm terribly sorry that I didn't see this PR before working on #55 which also implements this. Closing this as this is a duplicate in effect. Thank you!

**Comment by jezdez** [comment]
Actually, this has more than just the changes in .travis.yml. Reopening.

**Comment by shimizukawa** [comment]
Thanks!

**Pull request merged**